experiments/AtmosLES/taylor-green.jl

```diff
@@ -202,11 +204,22 @@ function main()
         init_on_cpu = true,
         Courant_number = CFL,
     )
-    dgn_config = config_diagnostics(driver_config)
-
+    #dgn_config = config_diagnostics(driver_config)
+    energy_file = "/home/yassine/Kinetic_energy.txt"
```

Comment:
We cannot merge such an absolute path.

Reply:
@kpamnany Good point, how would we best get around this ?

Reply:
@slifer50 We could permit write operations into the `--output-dir` similar to the VTK and NetCDF diagnostics (?)

Reply:
Yes, unfortunately I'm not sure how to do that.